Backpack/src/androidTest/java/net/skyscanner/backpack/button/TextMeasurementTest.kt

```diff
@@ -6,6 +6,7 @@ import com.nhaarman.mockitokotlin2.any
 import com.nhaarman.mockitokotlin2.eq
 import com.nhaarman.mockitokotlin2.mock
 import com.nhaarman.mockitokotlin2.verify
+import net.skyscanner.backpack.button.internal.TextMeasurement
```

Comment:
Let's move it to the same package instead

Reply:
Done